package.json

```diff
@@ -30,20 +35,22 @@
     "color-namer": "^1.4.0",
     "color-rgba": "^2.4.0",
     "events": "^3.3.0",
+    "fonteditor-core": "^2.1.11",
     "maplibre-gl": "^2.4.0",
     "openmapsamples": "github:adamfranco/OpenMapSamples",
     "openmapsamples-maplibre": "github:adamfranco/OpenMapSamples-MapLibre",
+    "sync-request": "^6.1.0",
```

Comment:
Node.js has `fetch` now, and for prior versions you can turn it on with a flag or use `undici` from npm.

Reply:
I removed sync-request in b8e6763.  It turns out it wasn't actually in use, so it must have crept in while I was evaluating alternatives.